src/plugins/data/public/query/query_string/query_string_manager.ts

```diff
@@ -61,7 +61,7 @@ export class QueryStringManager {
   }
 
   public getUpdates$ = () => {
-    return this.query$.asObservable();
+    return this.query$.asObservable().pipe(skip(1));
```

Comment:
Why was this necessary?

Reply:
This failing test pointed me to it: https://github.com/elastic/kibana/runs/1258846854

```
└-: usage of discover:searchOnPageLoad
             └-> "before all" hook
             └-> should fetch data from ES initially when discover:searchOnPageLoad is false
               └-> "before each" hook: global before each
               └- ✖ fail: discover app discover test usage of discover:searchOnPageLoad should fetch data from ES initially when discover:searchOnPageLoad is false
               │      Error: expected 1 to equal 0
               │       at Assertion.assert (packages/kbn-expect/expect.js:100:11)
               │       at Assertion.be.Assertion.equal (packages/kbn-expect/expect.js:227:8)
               │       at Assertion.be (packages/kbn-expect/expect.js:69:22)
               │       at Context.<anonymous> (test/functional/apps/discover/_discover.js:244:64)
               │       at process._tickCallback (internal/process/next_tick.js:68:7)
```


because `this.query$` is `BehaviorSubject` and not a regular `Subject`, this `getUpdates$` behaved differently comparing to similar observables in our other services. It always synchronously emitted current value: 


```
query.getUpdates$.subscribe(() => {
// before the fix this would always fire in the begging
}) 
```
 
I added a unit test: src/plugins/data/public/query/query_string/query_string_manager.test.ts